src/javascript/javascript-import-scanner.ts

```diff
@@ -39,7 +40,7 @@ export class JavaScriptImportScanner implements JavaScriptScanner {
             document.sourceRangeForNode(node.source)!, node));
       }
     });
-    return imports;
+    return {features: imports, warnings: []};
```

Comment:
for these and others, would it be safer to return `warnings: visitor.warnings` even if we're assuming today that visitor will have no warnings?